src/arrays/quarray.jl

```diff
@@ -34,6 +34,8 @@
     bases(qarr::QuArray, i) = qarr.bases[i]
     bases(qarr::AbstractQuArray) = ntuple(ndims(qarr), i->bases(qarr, i))
 
+    copy{B,T,N,A}(qa::QuArray{B,T,N,A}) = QuArray(copy(qa.coeffs), copy(qa.bases))
```

Comment:
@acroy I was just imitating the copy type parameters, I have observed that at some places it is used and others it is not though I could not understand the reason behind this. Any inputs on this would be helpful.